maven-plugin/src/main/java/org/openmrs/maven/plugins/Deploy.java

```diff
@@ -149,7 +149,13 @@ private void runInteractiveMode(Server server, ServerUpgrader upgrader) throws M
                         server.getName(),
                         server.getVersion()));
 
-                distro = wizard.promptForRefAppVersion(versionsHelper);
+                if (server.getName().equals("Platform") || server.getDistroGroupId() == null || server.getDistroArtifactId() == null) {
+                    // If its impossible to define distro, prompt refapp distro versions
+                    distro = wizard.promptForRefAppVersion(versionsHelper);
+                } else {
+                    // If its possible to define distro, prompt that distro's versions
+                    distro = wizard.promptForDistroVersion(server.getDistroGroupId(), server.getDistroArtifactId().replace("-omod",""), server.getVersion(), server.getName(), versionsHelper);
```

Comment:
There is a little problem since I've used org.openmrs.module:pihmalawi test case for this issue.
Problem is that because of this `distro`'s `groupId` is `org.openmrs.module` (instead of `org.openmrs.distro`) SDK saves its `artifactId` in `server.properties` as `pihmalawi-omod`.
Question is:
Do we want to support that "distro-module" artifacts by stripping `-omod`?